### PR TITLE
CT-129 Humanize user selected values in the email

### PR DESCRIPTION
**Pull request by aliuk2012**
https://dsdmoj.atlassian.net/browse/CT-129

### CODE REVIEW

File: app/views/correspondence_mailer/new_correspondence.html.slim [hunk 1]
```diff
@@ -1,12 +1,12 @@
 h2
-  = "New #{@correspondence.type}"
+  = "New #{@correspondence.type.humanize}"
```
Comment (misaka) [inline app/views/correspondence_mailer/new_correspondence.html.slim]: i18n?

File: app/views/correspondence_mailer/new_correspondence.html.slim [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 p
   = mail_to @correspondence.email, "#{@correspondence.name}(#{@correspondence.email})"
   |  submitted the following:
 
 
-h3 = "Topic: #{@correspondence.topic}"
+h3 = "Topic: #{@correspondence.topic.humanize}"
```
Comment (misaka) [inline app/views/correspondence_mailer/new_correspondence.html.slim]: i18n?

**Comment by misaka** [comment]
How easy/hard is it to i18n those?

**Comment by aliuk2012** [comment]
I don't think its too hard but I think we should cover it as part of the i18n ticket.

**Pull request merged**